dbms/src/Storages/DeltaMerge/Index/MinMaxIndex.cpp

```diff
@@ -397,7 +457,18 @@ RSResult MinMaxIndex::checkNullableGreaterEqual(size_t pack_index, const Field &
 {
     const auto & column_nullable = static_cast<const ColumnNullable &>(*minmaxes);
 
+    const auto & null_map = column_nullable.getNullMapColumn();
+    bool min_is_null = null_map.getElement(pack_index * 2);
+
+    // if min value is null, then it must be the old version of minmax index.
+    // In this case, we directly return RSResult::Some, because we can not know about the real min value.
+    if (min_is_null)
```

Comment:
Seems that this logic is not checked by the unit test, could you add some tests for it?